### ISSUE TITLE
Exposed Filter OL3

### ISSUE THREAD

**Issue by jlsevillano** [drupol/openlayers#31]
Making a GeoJSON Feed, add it to a map and  display the map block, if you try to filter by one field, it returns an error.

What's wrong?

**Comment by drupol** [comment]
How do you filter ? Through views ?

**Comment by jlsevillano** [comment]
Yeah. I follow this:
http://www.petejwhite.com/drupal-views-with-openlayers-3-0-beta1/

Then I have 1 view (geojson feed), and i want to exposed filter (for example thing all the countries in the world, and make a filter by continent). If i expose the filter in view, it returns a json and not display my map filtered.

**Comment by drupol** [comment]
This is not yet supported indeed...

**Comment by jlsevillano** [comment]
Then i can't filter... or i need to do it by another way?

**Comment by drupol** [comment]
Indeed, filtering is not yet supported. Best way to do that is to create multiple geojson feed, add them to the map and filter with a LayerSwitcher, like this: http://dev.through-my-eyes.be/openlayers_examples/openlayers_examples_map_ol3layerswitcher

**Comment by jlsevillano** [comment]
To my project is not a solution, a lot of combinations. I hope in the future we can add OL3, but after your explanitions i'm going to use OL2, it's a pitty. A lot of thanks Pol

**Comment by drupol** [comment]
Yes it's sad.
Unfortunately, I'm the only one to work on OL3, paches, pull requests, etc etc any help is welcome to improve the module.

**Comment by jlsevillano** [comment]
OK, if we'll find a solution we're going to tell you and help you.

**Comment by xmacinfo** [comment]
Any new developement about this request? I am trying to migrate from OL2 to OL3 and having no Views exposed filter will prevent us from moving forward.
